addons/hr_expense/models/hr_expense.py

```diff
@@ -659,6 +697,13 @@ def paid_expense_sheets(self):
     def refuse_sheet(self, reason):
         if not self.user_has_groups('hr_expense.group_hr_expense_user'):
             raise UserError(_("Only HR Officers can refuse expenses"))
+        elif self.employee_id.user_id.id == self.env.user.id and not self.user_has_groups('hr_expense.group_hr_expense_manager'):
```

Comment:
If we change code in approve_sheet I suppose we will do the same kind of changes here.